README.md

```diff
@@ -26,26 +39,41 @@ Download from [SublimeText 3 page](https://www.sublimetext.com/3)
 			[
 				"java",
 				"-jar",
-				"PATH/TO/camel-lsp-server-1.1.0-SNAPSHOT.jar"
+				"PATH/TO/camel-lsp-server-1.1.0.jar"
 			],
 			"enabled": true,
-			"languageId": "camel",
-			"scopes":
-			[
-				"text.xml"
+			"languages": [
+				{
+					"selector": "text.xml",
+					"priority_selector": "text.xml",
+				},
+				{
+					"selector": "source.java",
+					"priority_selector": "source.java",
+				}
 			],
-			"syntaxes":
-			[
-				"Packages/XML/XML.sublime-syntax"
-			]
-		}
-	}
+		},
+	},
+}
+```
+## Inside LSP-lemminx.sublime-settings for XML.
+- Tools -> Command palette... -> Prefernces:LSP-lemminx settings
+- Tools -> LSP:Enable package server lemminx both Globally and in Project.
+```
+{
+	"ignored_packages":["Vintage"]
+}
+```
+## Inside LSP-jdtls.sublime-settings for JAVA.
+- Tools -> Command palette... -> Prefernces:LSP-jdtls settings
+- Tools -> LSP:Enable package server jdtls both Globally and in Project.
+```
+{
+		"version": "1.12.0-202206011637",
+		"enabled": true,
 }
 ```
 
 Enjoy!
-![Completion of Camel URI in Sublime](sublimeCompletion.png "Completion of Camel URI in Sublime")
-
-## Current limitations
+Completion of Camel URI in Sublime.
```

Comment:
Sounds like a left over of removed image

Reply:
remains this one

Reply:
I have updated the changes now. Please review it